.gitignore

```diff
@@ -12,3 +12,4 @@
 *.o
 *.a
 mkmf.log
+*.swp
```

Comment:
This should go to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).